### DISCUSSION TITLE
How do I hide the powershell version in vscode?

### DISCUSSION THREAD

**Discussion by 404-xianjin** [JanDeDobbeleer/oh-my-posh#5748] [General]
![image](https://github.com/user-attachments/assets/25361a6e-2996-4c36-8db6-8b972e78dedd)

**Comment by JanDeDobbeleer** [comment]
You need to start PowerShell with the `-NoLogo` parameter.

**Reply by 404-xianjin** [reply]
Thanks ,Problem solved by -Nologo.
![image](https://github.com/user-attachments/assets/4a4949a2-9191-480f-80c2-4f006f3c6684)
![image](https://github.com/user-attachments/assets/006ed3ec-bfaa-4167-8552-c4e046262946)
